packages/mui-codemod/src/v5.0.0/base-remove-component-prop.test/actual.tsx

```diff
@@ -3,6 +3,8 @@ import MaterialInput from '@mui/material/Input';
 import Input from '@mui/base/Input';
 import Switch from '@mui/base/Switch';
 import Badge from '@mui/base/Badge';
+import PopperUnstyled from '@mui/base/Popper';
+import { Popper } from '@mui/base';
```

Comment:
```suggestion
import PopperUnstyled from '@mui/base/Popper';
import { Popper } from '@mui/base';
import { Popper as PopperBase } from '@mui/base';
```

Just to have this use-case as a test.